openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/flagencoders/WheelchairFlagEncoder.java

```diff
@@ -229,7 +329,8 @@ public int defineNodeBits(int index, int shift) {
                 	}
                 	// foot=no, restricted, private
                 	if (way.hasTag("foot", restrictedValues)) {
-                		return EncodingManager.Access.CAN_SKIP;
+                        if (DEBUG_SKIPPED_WAYS) LOGGER.warn(String.format(DEBUG_MSG_SKIPPED, "no pedestrian ferry", way.getId(), way.getTags().toString()));
```

Comment:
Do we really need a separate log-level hierarchy? (DEBUG_SKIPPED_WAYS in addition to LOGGER.*) This messes up the whole file.

Reply:
you're right, fixed